packages/tracker-core/SnesSession.ts

```diff
@@ -91,10 +92,15 @@ export class SnesSession {
   public async connect() {
     this.resetState();
 
+    // SNI changed the port it listens on. Check current and legacy ports
     if (!this._client) {
       this._client = new ws.w3cwebsocket(wsServer);
     }
 
+    if (!this._client) {
```

Comment:
shouldn't this be after the _connect attempt? I don't think it touches the port until then